plugins/provisioners/ansible/provisioner/guest.rb

```diff
@@ -65,15 +64,27 @@ def check_and_install_ansible
           end
         end
 
+        def execute_ansible_galaxy_on_guest
+          command_values = {
+            :ROLE_FILE => get_galaxy_role_file(config.provisioning_path),
+            :ROLES_PATH => get_galaxy_roles_path(config.provisioning_path)
+          }
+          remote_command = config.galaxy_command % command_values
```

Comment:
This is pretty cool :smile:

I would recommend we use lowercase names for the interpolated strings though; uppercase usually means a constant and could be misleading